persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/table/management/DB2TableManager.java

```diff
@@ -21,6 +21,15 @@
       super(connectionFactory, config, metaData, LOG);
    }
 
+   @Override
+   public String getInsertRowSql() {
+      if (insertRowSql == null) {
+         insertRowSql = String.format("INSERT INTO %s (%s,%s,%s) VALUES (?,?,?)", getTableName(),
+               config.idColumnName(), config.timestampColumnName(), config.dataColumnName());
```

Comment:
shouldn't the insert have the same order as upsert?

Reply:
It is, the only difference is the order of the `String.format(...)` params not the set values.